mmpretrain/datasets/categories.py

```diff
@@ -1141,3 +1141,308 @@
+    '/c/cottage_garden', '/c/courthouse', '/c/courtroom', '/c/courtyard',
+    '/c/covered_bridge/exterior', '/c/creek', '/c/crevasse', '/c/crosswalk',
+    '/c/cubicle/office', '/d/dam', '/d/delicatessen', '/d/dentists_office',
+    '/d/desert/sand', '/d/desert/vegetation', '/d/diner/indoor',
+    '/d/diner/outdoor', '/d/dinette/home', '/d/dinette/vehicle',
+    '/d/dining_car', '/d/dining_room', '/d/discotheque', '/d/dock',
+    '/d/doorway/outdoor', '/d/dorm_room', '/d/driveway',
+    '/d/driving_range/outdoor', '/d/drugstore', '/e/electrical_substation',
+    '/e/elevator/door', '/e/elevator/interior', '/e/elevator_shaft',
+    '/e/engine_room', '/e/escalator/indoor', '/e/excavation',
+    '/f/factory/indoor', '/f/fairway', '/f/fastfood_restaurant',
+    '/f/field/cultivated', '/f/field/wild', '/f/fire_escape',
+    '/f/fire_station', '/f/firing_range/indoor', '/f/fishpond',
+    '/f/florist_shop/indoor', '/f/food_court', '/f/forest/broadleaf',
+    '/f/forest/needleleaf', '/f/forest_path', '/f/forest_road',
+    '/f/formal_garden', '/f/fountain', '/g/galley', '/g/game_room',
+    '/g/garage/indoor', '/g/garbage_dump', '/g/gas_station',
+    '/g/gazebo/exterior', '/g/general_store/indoor',
+    '/g/general_store/outdoor', '/g/gift_shop', '/g/golf_course',
+    '/g/greenhouse/indoor', '/g/greenhouse/outdoor', '/g/gymnasium/indoor',
+    '/h/hangar/indoor', '/h/hangar/outdoor', '/h/harbor', '/h/hayfield',
+    '/h/heliport', '/h/herb_garden', '/h/highway', '/h/hill', '/h/home_office',
+    '/h/hospital', '/h/hospital_room', '/h/hot_spring', '/h/hot_tub/outdoor',
+    '/h/hotel/outdoor', '/h/hotel_room', '/h/house',
+    '/h/hunting_lodge/outdoor', '/i/ice_cream_parlor', '/i/ice_floe',
+    '/i/ice_shelf', '/i/ice_skating_rink/indoor',
+    '/i/ice_skating_rink/outdoor', '/i/iceberg', '/i/igloo',
+    '/i/industrial_area', '/i/inn/outdoor', '/i/islet', '/j/jacuzzi/indoor',
+    '/j/jail/indoor', '/j/jail_cell', '/j/jewelry_shop', '/k/kasbah',
+    '/k/kennel/indoor', '/k/kennel/outdoor', '/k/kindergarden_classroom',
+    '/k/kitchen', '/k/kitchenette', '/l/labyrinth/outdoor', '/l/lake/natural',
+    '/l/landfill', '/l/landing_deck', '/l/laundromat', '/l/lecture_room',
+    '/l/library/indoor', '/l/library/outdoor', '/l/lido_deck/outdoor',
+    '/l/lift_bridge', '/l/lighthouse', '/l/limousine_interior',
+    '/l/living_room', '/l/lobby', '/l/lock_chamber', '/l/locker_room',
+    '/m/mansion', '/m/manufactured_home', '/m/market/indoor',
+    '/m/market/outdoor', '/m/marsh', '/m/martial_arts_gym', '/m/mausoleum',
+    '/m/medina', '/m/moat/water', '/m/monastery/outdoor', '/m/mosque/indoor',
+    '/m/mosque/outdoor', '/m/motel', '/m/mountain', '/m/mountain_snowy',
+    '/m/movie_theater/indoor', '/m/museum/indoor', '/m/music_store',
+    '/m/music_studio', '/n/nuclear_power_plant/outdoor', '/n/nursery',
+    '/o/oast_house', '/o/observatory/outdoor', '/o/ocean', '/o/office',
+    '/o/office_building', '/o/oil_refinery/outdoor', '/o/oilrig',
```

Comment:
In any case, adding a first letter folder in front of the category name is very strange. It is best for the category name to be one word.